Bubble_Sort/Bubble_Sort.kt

```diff
@@ -0,0 +1,32 @@
+//Function for Bubble Sort
+fun bubbleSort(list: MutableList<Int>):List<Int> {
+    /* flag keeps track if any swap happened or not,
+     if swap not happened then array is sorted and it will break
+      out of the loop and will save time from
+      running for loop unnecessarily.*/
+    var flag: Int = 0
+    for(i in 0 until (list.size - 1)) {
+        for (j in 0 until (list.size - i - 1)) {
+            if(list[j] > list[j + 1]) {
```

Comment:
space after if

Reply:
Done

Reply:
You added space after the second if. Please add space before first one too.

Reply:
Done